addons/website_slides/controllers/main.py

```diff
@@ -699,18 +699,19 @@ def add_category(self, channel_id, name, **kw):
 
     #Not using the /web/dataset/resequence route as a slide can be dragged into another category at the same time
     @http.route('/slides/resequence_slides', type="json", website=True, auth="user")
-    def resequence_slides(self, slides_data=None, **kw):
-        channel_user = None
-        for slide in slides_data:
-            s = request.env['slide.slide'].browse(slide['id'])
-            if not channel_user:
-                channel_user = s.channel_id.user_id
-                if request.env.user != channel_user:
-                    return {'error': 'Only the responsible of the channel can edit it'}
-            s.write({
-                'sequence': slide['sequence'],
-                'category_id': slide['category_id']
-            })
+    def resequence_slides(self, slides_data=None):
```

Comment:
/slides/channel/resequence ?

Take channel_id and data for resequence. (other comments sould be updated accordingly, i have la flemme)

Reply:
Done.